releasenotes/notes/0.19/added-snapshot-tests-for-backend-mapping-functions-5961300e09f05be0.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Added the parameter  ``filename`` to :func:`~qiskit.visualization.plot_gate_map` and :func:`~qiskit.visualization.plot_coupling_map`, which allows saving the resulting images to a file.
```

Comment:
While there are no linewrap rules enforced here when we port this over to qiskit/qiskit in the release_notes.rst file a 100 character wide line length is enforced. So it's good to wrap these now so we don't have to manually fix them in the output of `reno report` that we embed in the release notes file.

Reply:
I reflowed all release notes that looked too long.